README.md

```diff
@@ -322,6 +323,25 @@ typedef NS_ENUM(NSInteger, NYTAdRequestState) {
 };
 ```
 
+## Boxing Enumerated Types
+
+Althought `enum`s are integral values, they cannot be directly used as [boxed literals](http://clang.llvm.org/docs/ObjectiveCLiterals.html#boxed-enums). So when creating a dictionary using `enum`s, they should be written as boxed literals instead of their corresponding `NSNumber` expression. 
+
+**For example:**
+```objc
+typedef NS_ENUM(NSInteger, NYTDaysOfTheWeek){
+     NYTMonday = 0, 
+     NYTTuesday
+}; 
+
+NSDictionary *daysOfTheWeek = @{@"monday" : @(NYTMonday), @"tuesday" : @(NYTTuesday)};
```

Comment:
I would as well, but it's not currently in place under the [Literals](https://github.com/NYTimes/objective-c-style-guide#literals) section of the guide